apps/ui-library/content/docs/nextjs/social-auth.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Social Authentication
+description: Social authentication block for Next.js
+---
+
+<BlockPreview name="social-auth/auth/login" />
+<Callout>
+  The block is using Github provider by default, but can be easily switched by changing a single
+  parameter.
+</Callout>
+
+## Installation
+
+<BlockItem name="social-auth-nextjs" description="All needed components for the social auth flow" />
+
+## Folder structure
+
+This block includes the [Supabase client](/ui/docs/nextjs/client). When installing, you can skip overwriting it.
```

Comment:
This block includes the Supabase client. If you already have one installed, you can skip overwriting it.